server/backend/database/database.go

```diff
@@ -118,6 +124,9 @@ type Database interface {
 	// FindUserInfo returns a user by the given username.
 	FindUserInfo(ctx context.Context, username string) (*UserInfo, error)
 
+	// FindUserInfoById finds a user by the given id.
+	FindUserInfoByID(ctx context.Context, id string) (*UserInfo, error)
```

Comment:
We use `types.ID` to identify the resource in DB. Is it possible to change this?

```suggestion
	FindUserInfoByID(ctx context.Context, id types.ID) (*UserInfo, error)
```

Reply:
Initially, it was implemented based on `types.ID`, but when testing in memoryDB, finding user by clientID with type `types.ID` did not work correctly. 

This was solved by converting `types.ID` to `string` when querying memoryDB.